tests/v2/integration/steveapi/steve_api_test.go

```diff
@@ -159,7 +159,7 @@ func (s *SteveAPITestSuite) TearDownSuite() {
 }
 
 func (s *SteveAPITestSuite) SetupSuite() {
```

Comment:
Since the URL changed, the `formatJSON` on 918 will have to parse out the new URL and replace it with a generic string to avoid the JSON examples changing every run.

Reply:
Does that also mean running the tests to have the new generic string/url be updated in the JSON examples and committing that as well? What about the output.csv?

Reply:
Yes, the JSON examples and csv will need to be updated and committed

Reply:
So I added a new regex, and replaced the urls with `rancherurl`, and updated the JSON, README, and csv